.github/actions/publish-docs/action.yml

```diff
@@ -5,18 +5,15 @@ inputs:
   workspace_path:
     description: 'The workspace to publish docs for'
     required: true
+  token:
+    description: 'Token to use for publishing.'
+    required: true
 runs:
   using: composite
   steps:
-    # We put the github pages in another directory.
-    - uses: actions/checkout@v3
+    - uses: launchdarkly/gh-actions/actions/publish-pages@publish-pages-v1.0.1
+      name: 'Publish to Github pages'
       with:
-        path: gh-pages
-        ref: gh-pages
-    - name: Publish Docs
-      shell: bash
-      run: |
-        echo "Publishing docs for: $WORKSPACE"
-        ./scripts/publish-doc.sh $WORKSPACE
-      env:
-        WORKSPACE: ${{ inputs.workspace_path }}
+        docs_path: ${{ inputs.workspace_path }}
```

Comment:
I am reasonably sure this should be in a subdirectory. `${{ inputs.workspace_path }}/docs` 

The output in the doxyfile contains `docs` and the old script used `docs`.

Reply:
I did that originally (first commit), and it then broke all existing docs links (like from README + LaunchDarkly public docs).

Reply:
Not including it means you are publishing all the source into docs which happens to include docs. If you want it in a sub-directory on the output, then put that into the output path as well.

Reply:
![image](https://github.com/launchdarkly/cpp-sdks/assets/4955475/702cea21-5e97-44a5-8847-489be9c7d34c)

Reply:
What it used to, and correctly, looked like:
![image](https://github.com/launchdarkly/cpp-sdks/assets/4955475/098d408e-8c5f-4a7a-bceb-19871c22a9e4)

Reply:
So, before we took the $workspace/docs and put them into $workspace/docs.

Currently you are putting $workspace/ which happens to contain docs into $workspace/.

Reply:
Is this what you tried?
```
docs_path: ${{ inputs.workspace_path }}/docs
output_path: ${{ inputs.workspace_path }}/docs
```

If it doesn't work, then we need to fix a bug in the action.

Reply:
Ok, I had only added `/docs` to the `docs_path`. That was the problem. Updated both and all is working now (+ I've fixed up the `gh-pages` branch.)